brainglobe_segmentation/tracks/analysis.py

```diff
@@ -103,6 +104,17 @@ def run_track_analysis(
     return spline
 
 
+def get_distances(spline, voxel_size=10):
+    """
+    For a given spline, calculate the distance between each point
```

Comment:
```suggestion
    For a given spline, calculate the distance between each point.
    Assumes a customisable isotropic voxel size (default 10) in microns.
```
Be clear in docs about this assumption?